### ISSUE TITLE
Extension pack view makes actual readme content very small when the editor group is small

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#166627]
![Recording 2022-11-17 at 11 22 42](https://user-images.githubusercontent.com/2193314/202539281-7fa6cda6-6297-429f-8241-4b3944861ed2.gif)

**Comment by sandy081** [comment]
An extension pack has to be light weight and only contain extensions, it is not expected to be a feature rich extension and also have some goodies. I would recommend Jupyter extension to uncategorize it as extension pack

**Comment by Tyriar** [comment]
Sure, I agree. However there is still an issue here as extension packs also have descriptions. This is using my typical editor size:

![Recording 2022-11-22 at 08 22 54](https://user-images.githubusercontent.com/2193314/203367249-cf6b2b71-4ce9-4bef-b934-a8acdb83c660.gif)

**Comment by sandy081** [comment]
Agreed. Probably providing an horizontal sash is helpful or having max height of extension pack section to be two rows

**Comment by Tyriar** [comment]
This is worth than expected, it's not possible to read the description on small windows since the body doesn't have a scroll bar:

<img width="577" alt="Screenshot 2023-08-29 at 9 24 35 AM" src="https://github.com/microsoft/vscode/assets/2193314/74f4065f-9d42-4d92-b050-ee9c22d406db">

**Comment by sandy081** [comment]
Extension pack content height is now changed according to the overall height. As the overall height reduces the extension pack content height is reduced to min of one row. 

Note: Read me still cant be shown if the overall height is less than an extension pack row height.
